communications/README.md

```diff
@@ -23,7 +23,7 @@ Minimize Distractions: Nowadays we have way too many channels of receiving infor
 
 Opt-out: Leave channels that you are no longer interested in or obligated to.
 
-Asynchronous Communication by Default: Whenever you want to communicate with others, default to an asynchronous way. By doing so, you respect your coworker's time and their productivity and the fact that they may be in the "zone" and should not be interrupted. 
+Asynchronous Communication by Default: Whenever you want to communicate with others, default to an asynchronous way. By doing so, you respect your coworker's time and their productivity and the fact that they may be in the "zone" and should not be interrupted.
```

Comment:
I just remembered that we could address this point a little differently. Maybe mention something about how some people prefer in-person chats. Suggestion:
'Asynchronous Communication by Default: Each person has their own communication preferences. However, unless the person has stated that they prefer in-person chats over asynchronous communication, default to an asynchronous way. By doing so....'